README.md

```diff
@@ -69,10 +80,16 @@ $ yarn add fabric
 ```
 
 ```js
-// es6 imports
+// v6 es
+import * as fabric from 'fabric'; // browser
```

Comment:
Let's also show the `import { Canvas, Rect } from 'fabric'`

Reply:
done